channels/nz.m3u

```diff
@@ -1,27 +1,33 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="TheBreeze.nz" tvg-name="Breeze TV NZ" tvg-country="NZ" tvg-language="English" tvg-logo="https://i.imgur.com/3dpH9XF.jpg" group-title="Music",Breeze TV (720p)
+https://livestreamdirect-breezetv.mediaworks.nz/breezetv.m3u8
+#EXTINF:-1 tvg-id="TheEdge.nz" tvg-name="The Edge NZ" tvg-country="NZ" tvg-language="English"  tvg-logo="https://i.imgur.com/ZHJVvOj.png" group-title="Music",The Edge (720p)
```

Comment:
No need to keep the "NZ" next to the name of the channel. The channel to itself is well located :)